examples/asr/configs/quartznet_an4.yaml

```diff
@@ -1,5 +1,5 @@
 model: "QuartzNet-AN4"
-sample_rate: &sr 16000
```

Comment:
Why was sample_rate removed?

Reply:
Previously, passing this file as a config to `quartznet.py` would cause an error since this top-level sample_rate was passed in as well as the ones in the data layer params (i.e. duplicate sample_rate params passed in).
I just removed the two duplicate sample_rate params causing the issue below.

Reply:
Ok